### ISSUE TITLE
parallex effect homepage

### ISSUE THREAD

**Issue by tanmoy27112000** [tanmoy27112000/flutter_web_base#4]

**Comment by viendimine** [comment]
Hi tanmoy27112000 , Myself Vishap raj , a Second Undergraduate Student Currently persuing Btech in Electrical Engineering From
prestigious Indian Institute Of Technology Kanpur , I want to contribute in this issue, my motivation to contribute in this issue is that as I have already done a project of Flutter Development and so I am Very Familiar to it and also I will enhance my knowldege by implementing Parallex widget in your project's homepage .Thanks,Waiting for a positive response.

**Comment by tanmoy27112000** [comment]
@vishapraj sure you can work on it

**Comment by viendimine** [comment]
Thanks tanmoy27112000

**Comment by tanmoy27112000** [comment]
Clearing assignment for now will reassign in after the event starts

**Comment by tanmoy27112000** [comment]
@vishapraj I have assigned you the ticket, First before starting, confirm the actual design that you will be building.

**Comment by avirupnandi1** [comment]
Hi @tanmoy27112000 ..If this issue is open for assigning, I can contribute in it!

**Comment by tanmoy27112000** [comment]
@avirupnandi1 sure you can take it up
